BankManagerConsoleApp/BankManagerConsoleApp/Node.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Node.swift
+//  BankManagerConsoleApp
+//
+//  Created by 변재은 on 2022/06/28.
+//
+
+import Foundation
+
+class Node<T> {
+    let value: T
+    var next: Node<T>?
+    
+    init(value: T, next: Node? = nil) {
```

Comment:
```suggestion
    init(value: T, next: Node<T>? = nil) {
```
이런건 PR 올리기전에 훑어보면 찾으실수 있을꺼에요!